### DISCUSSION TITLE
Offline mode support

### DISCUSSION THREAD

**Discussion by FurblandChannel** [Lea-fish/Leafish#220] [Ideas]
please i migrated and cant log in heeeeeelp

**Comment by phoebehfharris** [comment]
Seconded. I have no way of running Leafish at all since my Mojang account went and left for Microsoft

**Reply by terrarier2111** [reply]
Offline accounts are already supported you need to enable a setting in your config and then you can insert an account without password and you got yourself an offline account.

**Reply by phoebehfharris** [reply]
I'm so sorry, I didn't read this until now. That's helpful! Is there any reason this is hidden behind a config?

**Comment by terrarier2111** [comment]
This is already solved as explained by my comment

**Reply by phoebehfharris** [reply]
What setting would you have to enable?

**Reply by andreaf74** [reply]
https://github.com/Lea-fish/Leafish/issues/49#issuecomment-1621763938
